crl/updater/updater.go

```diff
@@ -21,6 +23,13 @@ import (
 	sapb "github.com/letsencrypt/boulder/sa/proto"
 )
 
+// startTime is used as a universal starting point against which other times can
+// be compared. This time must be less than 290 years (2^63-1 nanoseconds) in
+// the past, to ensure that Go's time.Duration can represent that difference.
+// The significance of 2015-06-04 11:04:38 UTC is left as an exercise to the
+// reader.
+var startTime = time.Date(2015, time.June, 04, 11, 04, 38, 0, time.UTC)
```

Comment:
Can you make this a func so we know it doesn't change?

Reply:
Good call, done. I wish Go had a way to declare consts that aren't a primitive type.